src/Haskell/Ide/Engine/LSP/Config.hs

```diff
@@ -34,3 +42,20 @@ instance FromJSON Config where
 --   , _params = DidChangeConfigurationParams
 --                 {_settings = Object (fromList [("languageServerHaskell",Object (fromList [("hlintOn",Bool True)
 --                                                                                          ,("maxNumberOfProblems",Number 100.0)]))])}}
+
+instance ToJSON Config where
+  toJSON (Config h m l) = object [ "languageServerHaskell" .= r ]
+    where
+      r = object [ "hlintOn"             .= h
+                 , "maxNumberOfProblems" .= m
+                 , "liquidOn"            .= l
+                 ]
```

Comment:
Can we add a unit test showing that `FromJSON` and `ToJSON` are symmetrical? I often run into bugs when I only change one of the instances and not the other

Reply:
Nevermind, I read further and the test seems to be present already

Reply:
Why does this need a `ToJSON` instance? Not opposed to it just curious

Reply:
For the quickcheck roundtrip tests